src/halmos/__main__.py

```diff
@@ -511,7 +513,8 @@ def run_target_contract(
         ValueError: If the contract name cannot be found for the given address.
     """
     args = ctx.args
-    target_senders = ctx.target_senders
+    excluded_senders = ctx.excluded_senders
+    effective_target_senders = ctx.target_senders - excluded_senders
```

Comment:
it seems like we could shove `effective_target_senders` in the ContractContext in `setup_invariant_test_context` after these, right?

```
        process_target_senders(ctx, setup_ex)
        process_target_contracts(ctx, setup_ex)
        process_target_selectors(ctx, setup_ex)

        process_excluded_senders(ctx, setup_ex)
        process_excluded_contracts(ctx, setup_ex)
        process_excluded_selectors(ctx, setup_ex)
```